go/runtime/host/sandbox/bwrap_test.go

```diff
@@ -0,0 +1,43 @@
+package sandbox
+
+import (
+	"bytes"
+	"io"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestBubbleWrapSandbox(t *testing.T) {
+	// Skip test if there is no bwrap binary available.
+	if _, err := os.Stat(bwrapPath); os.IsNotExist(err) {
+		t.Skip("skipping as bubblewrap not available")
+	}
+
+	t.Run("BindData", testBwrapBindData)
+}
+
+func testBwrapBindData(t *testing.T) {
+	require := require.New(t)
+
+	// Bind some data from a buffer and run /bin/cat inside the sandbox to make sure it's there.
+	var stdout bytes.Buffer
+	p, err := NewBubbleWrap(Config{
```

Comment:
this is mostly duplicated with the `naked_test` should this test instead just run the naked test with bwrap sandbox?

Reply:
Done.